test_scripts/API/Registration/012_Registered_app_used_systemSoftwareVersion_value_ccpu_version_from_GetSystemInfo_during_one_ignition_cycle.lua

```diff
@@ -0,0 +1,56 @@
+---------------------------------------------------------------------------------------------------
+-- Regression check
+-- Use case:TBD
+--
+-- Requirement summary:
+-- TBD
+--
+-- Description:
+-- Check that SDL used for param systemSoftwareVersion value "ccpu_version" from "GetSystemInfo" during one ignition cycle
+-- In case:
+-- 1) Mobile application is registered.
+-- 2) Mobile app is re-registered.
+-- SDL does:
+-- 1) Successfully re-register application.
+-- 2) Not send GetSystemInfo during RegisterAppInterface.
+-- 3) Used for param systemSoftwareVersion value "ccpu_version" from "GetSystemInfo" during one ignition cycle
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/Registration/commonRAI')
+local hmi_values = require('user_modules/hmi_values')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local systemSoftwareVersionValue = "FORD"
+
+local valueForResponse = {
+  systemSoftwareVersion =  systemSoftwareVersionValue
+}
+
+--[[ Local Functions ]]
+local function getHMIValuesDuringFirstRAI()
+  local params = hmi_values.getDefaultHMITable()
+    params.BasicCommunication.GetSystemInfo.params.ccpu_version = "FORD"
+    params.BasicCommunication.GetSystemInfo.params.language = "EN-US"
+    params.BasicCommunication.GetSystemInfo.params.wersCountryCode = "user_wersCountryCode"
```

Comment:
@GetmanetsIrina Why do we need to update `language` and `wersCountryCode` parameters? According to description only `ccpu_version` needs to be updated.